src/helpers/contacts.js

```diff
@@ -125,9 +127,15 @@ export const harmonizeAndSortByFamilyNameGivenNameEmailCozyUrl = (
     updateIndexFullNameAndDisplayName(contact)
   )
   const concatedContacts = contactsWithIndexes.concat(updatedContacts)
-  const sortedContacts = sortBy(concatedContacts, [
+  const {
+    itemsFound: [mySelf],
```

Comment:
Sure that there is only one `mySelf` ?

Reply:
Yes "mySelf" here corresponds to `contact.me`, which is an attribute defined only on the Cozy user.